### DISCUSSION TITLE
Uyuni Administrator checkbox greyed out for user

### DISCUSSION THREAD

**Discussion by jchristopherlittle** [uyuni-project/uyuni#9219] [Q&A] [answered]
I'm adding my first user to a non-containerized Uyuni install (2024.07). The option to make the user an Uyuni admin, but the checkbox is greyed out and unable to be selected.

**Comment by mcalmer** [comment, marked as answer]
One of our oldies.
In that UI you cannot create an admin as Uyuni admin.
Go to "Admin" => "Users" and you will get a list of users. 
The column "Uyuni admin?" show the status and you can click there to make a user an Uyuni admin.
(This does not work for Org Admin - that can be changed in the other UI)

**Comment by jchristopherlittle** [comment]
Got it! Thank you!

Question: Why is this so unintuitive? Why have a greyed out field if its never to be used? If this is the accepted way of adding the capability to a user, why not add a link instead of the field (or just make the field usable)?

**Reply by mcalmer** [reply]
This come from the old spacewalk times. We just had no time yet to change this. Any help here is welcome.
